packages/zowe-explorer/CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Added a `zowe.settings.socketConnectTimeout` setting to provide users with the ability to specify a connection timeout for REST requests. [#3379](https://github.com/zowe/zowe-explorer-vscode/issues/3379)
 - Added a `zowe.settings.requestTimeout` setting to provide users with the ability to specify a request timeout for REST requests. [#3631](https://github.com/zowe/zowe-explorer-vscode/pull/3631)
 - Updated Zowe SDKs to `8.20.0` for technical currency. [#3631](https://github.com/zowe/zowe-explorer-vscode/pull/3631)
+- Added support for asynchronous operations. [#3640](https://github.com/zowe/zowe-explorer-vscode/issues/3640)
 
 ### Bug fixes
 
+- BugFix: Allowed multiple commands to be cancelled in succession. [#3640](https://github.com/zowe/zowe-explorer-vscode/issues/3640)
```

Comment:
where's the bug explanation? we should be describing what the bug was, besides describing the fix.

Reply:
resolved in c68cd4e86